src/wiki/keyboard-shortcuts.md

```diff
@@ -0,0 +1,208 @@
+| Alt + Page Up                                         | Moves up one screen.                                                                      |
+| Alt + Page down                                       | Moves down one screen.                                                                    |
+| Ctrl + F4                                             | Closes the active window.                                                                 |
+| Ctrl + A                                              | Selects all elements in a document or window.                                             |
+| Ctrl + D (or del key)                                 | Deletes the selected item an move it.                                                     |
+| Ctrl + E                                              | Opens Search option. (if available)                                                       |
+| Ctrl + R (or F5)                                      | Refreshes the active window.                                                              |
+| Ctrl + Y                                              | Redo an action.                                                                           |
+| Ctrl + Right arrow key                                | Moves the cursor to the beginning of the next word.                                       |
+| Ctrl + Left arrow key                                 | Moves the cursor to the beginning of the previous word.                                   |
+| Ctrl + Down arrow key                                 | Moves the cursor to the beginning of the next paragraph.                                  |
+| Ctrl + Up arrow key                                   | Moves the cursor to the beginning of the previous paragraph.                              |
+| Ctrl + Alt + Tab                                      | Switch between opened apps with arrow keys. (You can let go of the alt and tab keys)      |
+| Alt + Shift + Arrow keys                              | When a group or tile is selected on the Start menu, move it in the specified direction.   |
+| Ctrl + Shift + Arrow keys                             | When a tile is selected on the Start menu, drag it onto another tile to create a folder.  |
+| Ctrl + Arrow keys (to move to an element) + Spacebar  | Select multiple individual items within a window or on the desktop.                       |
+| Ctrl + Esc                                            | Opens Start Menu.                                                                         |
+| Ctrl + Shift + Esc                                    | Opens Task Manager.                                                                       |
+| Ctrl + Shift                                          | Changes the keyboard layout when multiple layouts are available.                          |
```

Comment:
Isn't this Alt+Shift or Win+Space?

Reply:
Which shortcut you're referring to?

Reply:
Ctrl+Shift, the one that "supposedly" changes keyboard layouts. Either of those that I mentioned are correct.

Reply:
I verified the shortcut, and it appears Alt + Shift is the correct one. I'll fix that right away.